### PR TITLE
ExecutionClosure#started_at

### PR DESCRIPTION
**Pull request by derwiki**

### CODE REVIEW

File: flytekit/models/execution.py [hunk 1]
```diff
@@ -281,15 +281,17 @@ def from_flyte_idl(cls, pb):
 
 class ExecutionClosure(_common_models.FlyteIdlEntity):
 
-    def __init__(self, computed_inputs, phase, error=None, outputs=None):
+    def __init__(self, computed_inputs, phase, started_at, error=None, outputs=None):
         """
         :param flytekit.models.literals.LiteralMap computed_inputs:
         :param int phase: From the flytekit.models.core.execution.WorkflowExecutionPhase enum
+        :param datetime started_at:
```
Comment (matthewphsmith) [inline flytekit/models/execution.py]: nit: `datetime.datetime`

File: flytekit/models/execution.py [hunk 2]
```diff
@@ -329,6 +338,7 @@ def to_flyte_idl(self):
         return _execution_pb2.ExecutionClosure(
             computed_inputs=self.computed_inputs.to_flyte_idl(),
             phase=self.phase,
+            start_at=self.started_at,
```
Comment (matthewphsmith) [inline flytekit/models/execution.py]: `started_at`, i think?

File: flytekit/models/execution.py [hunk 3]
```diff
@@ -349,6 +359,7 @@ def from_flyte_idl(cls, pb2_object):
             error=error,
             outputs=outputs,
             phase=pb2_object.phase,
+            started_at=pb2_object.started_at,
```
Comment (matthewphsmith) [inline flytekit/models/execution.py]: Protobuf timestamps are a bit annoying in converting to/from python datetimes. See: 

https://github.com/lyft/flytekit/blob/d4cfedc4c580f08bf904e6e474a0b948a4608737/flytekit/models/literals.py#L136

https://github.com/lyft/flytekit/blob/d4cfedc4c580f08bf904e6e474a0b948a4608737/flytekit/models/literals.py#L152

Comment (derwiki) [reply flytekit/models/execution.py]: Updated, modeled after:
https://github.com/lyft/flytekit/blob/d4cfedc4c580f08bf904e6e474a0b948a4608737/flytekit/models/node_execution.py#L75
https://github.com/lyft/flytekit/blob/d4cfedc4c580f08bf904e6e474a0b948a4608737/flytekit/models/node_execution.py#L89

**Comment by derwiki** [comment]
PTAL @matthewphsmith

File: README.md [hunk 4]
```diff
@@ -72,6 +72,7 @@ Flytekit is Python 2.7+ compatible, so when feasible, it is recommended to test
 virtualenv ~/.virtualenvs/flytekit2
 source ~/.virtualenvs/flytekit2/bin/activate
 python -m pip install -r requirements.txt
+python -m pip install -U .[all]
```
Comment (matthewphsmith) [inline README.md]: ahhh facepalm, thanks!!!

**Comment by derwiki** [comment]
:rocket:

**Pull request merged**